server/README.md

```diff
@@ -0,0 +1,80 @@
+# API
+<!-- Endpoints are meant to be preceded by /api (/operations == /api/operations)-->
+## Bookings
+- GET `/bookings`
+  - Param: Time frame (start date and end date)
+  - Response: List of all bookings
+  ##### NOTE: This endpoint is intended to be used for statistics 
+
+## Lectures
+- GET `/lectures`
```

Comment:
The returned lectures depend on the logged in user right? In that case it should probably be added to the documentation.
When you are logged in as a support officer or booking manager you are going to get all the lectures for example, while students and teacher get only their courses lectures.
Do you think would be useful to add a `from` and `to` filter? So that we can request lectures for a user in a specific period of time instead of getting all of them? Something like GET `/lectures?from=<start_date>&to=<end_date>`

Reply:
Ah okay I just reached the `students` endpoint, I get it. It makes sense.
The part about `from` and `to` filter could still be interesting tough.

Reply:
OK, I just inserted it in the student endpoint. As you suggest, it can be useful to have a GET `/lectures?from=<start_date>&to=<end_date>` directly.